pontoon/base/static/css/style.css

```diff
@@ -1408,6 +1408,10 @@ body > form,
   display: none;
 }
 
+#helpers > section ul li > p.translation-diff {
+  white-space: pre-wrap;
+}
+
```

Comment:
This is now being fixed in https://github.com/mozilla/pontoon/pull/837, so it can be removed. It turned out to be a separate bug anyways.